test/ci-cd/terraform/README.md

```diff
@@ -0,0 +1,43 @@
+# accelerated-platforms CI/CD Terraform
+
```

Comment:
Consider adding a 1-2 sentences description of what this is.

Reply:
Added